Admin/FieldDescription.php

```diff
@@ -16,7 +16,7 @@
 class FieldDescription extends BaseFieldDescription
 {
     /**
-     * {@inheritdoc}
+     * Constructor.
```

Comment:
I would completely remove the phpdoc here, saying that __construct id the constructor doesn't add anything.